lib/bambora/rest/batch_payment_file_upload_client.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+module Bambora
+  module Rest
+    class BatchPaymentFileUploadClient < Bambora::Rest::Client
+      CONTENT_DISPOSITION = 'form-data'
+      CONTENT_TYPES ={
```

Comment:
not critical but space for an amused guy `={` w/ mustache is missing :)

Reply:
🙃